vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -1,41 +1,29 @@
-# Get started with your first dashboard
+# Explore Vizro
 
-This tutorial guides you step-by-step through the process of creating a dashboard.
+This tutorial walks through the process of creating a sophisticated dashboard. It introduces Vizro components and explains how to configure a layout to suit the functionality you need. The example uses the [gapminder data](https://plotly.com/python-api-reference/generated/plotly.express.data.html#plotly.express.data.gapminder).
 
-By the end of this tutorial, you will have gained a deep understanding of Vizro dashboards: their structure, adding
-functionality, and configuring layouts to suit your specific needs.
+If you haven't yet done so, you may want to review the [first dashboard tutorial](../tutorials/first-dashboard.md) before starting on this one.
 
-This step-by-step guide is designed to equip you
-with the knowledge and skills required to create your own dashboards using Vizro.
+## 1. Install Vizro and get ready to run your code
 
-In case you are completely new, you may want to review our [Get started](../tutorials/first-dashboard.md) tutorial first.
+To get started with this tutorial, make sure you have [installed Vizro](../user_guides/install.md), and can run the dashboard code [within a Jupyter Notebook cell](../first_dashboard/#2-open-a-jupyter-notebook), or from a Python script.
 
-## Let's get started!
+## 2. Create a first dashboard page
 
-### 1. Install Vizro and get ready to run your code
+In this section we create a new [`Page`][vizro.models.Page] called `first_page`.
 
-In order to get started with this tutorial, you should have Vizro installed, and be ready to evaluate your code, either by
-pasting it into a Jupyter notebook cell, or by running it from a python script.
+The foundation of every Vizro dashboard is a [`Page`][vizro.models.Page] object. A page uses a set of component types to display the content of the page. These components can be objects such as [`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table], [`Card`][vizro.models.Card], [`Button`][vizro.models.Button], [`Container`][vizro.models.Container], or [`Tabs`][vizro.models.Tabs].
```

Comment:
Shall we just link to the overview page of components instead of listing them out separately here? Otherwise this might get out of sync quickly